.github/workflows/app.js.yml

```diff
@@ -31,6 +31,9 @@ jobs:
       - name: App Format Check
         working-directory: ./template
         run: npm run format:check
+      - name: App Typescript Check
+        working-directory: ./template
+        run: npm run typecheck
```

Comment:
Isn't this already being run under `yarn lint`?

@wespickett thought on this?

Reply:
Ok dope, so if that's the case we should have the `yarn check` command, use `typecheck` as well.